smac/intensification/successive_halving.py

```diff
@@ -939,26 +955,16 @@ def _top_k(self,
         top_configs = configs_sorted[:k]
         return top_configs
 
-    def _count_remaining_instances_for_stage(
+    def _count_remaining_tasks_for_stage(
```

Comment:
Maybe we should rename this to `_count_remaining_inst_seed_pairs_for_stage` to be more in SMAC lingo